src/Languages/Xml/Patterns/XmlCommentPattern.php

```diff
@@ -19,13 +19,14 @@
             foo
             -->'
 )]
+#[PatternTest(input: '<!-- foo --><div class="wrapper"><!-- foo2 -->', output: ['<!-- foo -->','<!-- foo2 -->'])]
 final readonly class XmlCommentPattern implements Pattern
 {
     use IsPattern;
 
     public function getPattern(): string
     {
-        return '(?<match>\<!--(.|\n)*-->)';
+        return '/(?<match>\<!--(.|\n)*-->)/mU';
```

Comment:
TIL about the `U` flag! Thanks :) 

You could also write it like this, I believe `(.|\n)*?` but `U` is actually nicer :)